docs/library/network.rst

```diff
@@ -210,6 +210,31 @@ For example::
 
         Disconnect from the currently connected wireless network.
 
+    .. method:: wlan.scan(cb)
+
+        Initiate scanning for the available wireless networks.
+
+        Scanning is only possible if the radio is in station or station+AP mode; if
+        called while in AP only mode, an exception will be raised.
```

Comment:
Should probably mention that the exception is an OSError (so that users can catch it if needed).